.github/workflows/formatting.yaml

```diff
@@ -41,3 +54,34 @@ jobs:
       if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }} 
       with:
         repo-token: ${{secrets.GITHUB_TOKEN}}
+
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2 
+    - name: Use Java 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: "8"
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
interesting that here it's 12.x and above it's 10.x, not sure why? Wondering in general if we have to keep this updated or if there's some value we can pull it from.

Reply:
Not a big deal though I suppose.

Reply:
That was a mistake haha